File: documentation/user/en/query/filtering/examples/comparable/attribute-is-not-null.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeIs("catalogNumber", NOT_NULL)
				),
				require(
					entityFetch(
						attributeContent("code", "catalogNumber")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/comparable/attribute-is-not-null.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeIs("catalogNumber", NotNull)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "catalogNumber")
        		)
        	)
        )
	)
);